components/header.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
```

Comment:
the file name should also start with a capital letter `Header.js`

Reply:
The function name inside is "Header"
Could this lead to problems, maybe?

Reply:
No, that should not cause any issues